index.html

```diff
@@ -60,7 +72,7 @@ <h2>Translate Raw Data Back Into a Form</h2>
         </p>
     </div>
 
-    <footer>Stylesheets and code are availabe open-source on <a href="{{ site.github.repository_url }}">GitHub</a>.
+    <footer>Built with ♡ by the <a href="https://innovation.nj.gov/">New Jersey Office of Innovation</a>.<br />Stylesheets and code are available open-source on <a href="{{ site.github.repository_url }}">GitHub</a>.
```

Comment:
💨 Is `br` the right tag here? Honestly I am not sure, I feel like "developers" try to avoid it, but I don't know if that's true, nor do I know why if it is true.

Reply:
Interesting point! This is the first time I've heard of it, but I found this stackoverflow thread that gave me more context: https://stackoverflow.com/questions/1726073/is-it-sometimes-bad-to-use-br. 

I personally don't see it as a big issue and prefer not adding additional divs to the footer just to format the line breaks.